src/main/java/com/booking/service/service/impl/UserServiceImpl.java

```diff
@@ -19,6 +20,15 @@ public void add(User user) {
         userDao.add(user);
     }
 
+    @Override
+    public User get(Long id) {
+        Optional<User> optionalUser = userDao.get(id);
+        if (optionalUser.isEmpty()) {
+            return null;
+        }
+        return optionalUser.get();
```

Comment:
1) you can do this check inline with optional;
2) It's better to throw an exception(you can use orElseThrow for this)